### PR TITLE
Hit the API

### PR DESCRIPTION
**Pull request by Koshao-sys**
In this pull request I have implemented the following requirements. 💯 

- [x] Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
- [x] Create a new game with the name of your choice by using the API.
- [x] Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
- [x] Use arrow functions instead of the function keyword.
- [x] Implement the form "Submit" button that will save a score for a game created by you (sending data to the API).
- [x] Use async and await JavaScript features to consume the API.
- [x] No styling is required.

### CODE REVIEW

File: src/modules/scores-api.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  }
+};
+
+const postScores = async (obj) => {
+  try {
+    const data = await postData(baseUrl + scoresEndpoint, obj, 'POST');
+    return { success: true, data };
+  } catch (error) {
+    return { success: false, error };
+    // console.log(error);
```
Comment (ichala) [inline src/modules/scores-api.js]: - [ ] to keep your code clean , kindly remove any commented code from ur repo 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Used Arrow functions ✔️
- Api Works ✔️


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/scores-api.js (hunk 1 above)
Comment (youmari) [inline src/modules/scores-api.js]: - [ ] it is bad to leave `console.log` in the production code. Console.log statements are used for debugging purposes and should be removed before the code is deployed to production. Leaving `console.log `statements in the production code can cause performance issues and can also expose sensitive information. so please remove it as the previous reviewer asked

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Koshao-sys  

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights 
🏁 PRO PR
🏁 No linter error 
## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Koshao-sys, 

To highlight 👍🏽 
✔️ No commented code
✔️ No linter errors

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**